scripts/requests/repositories.http

```diff
@@ -32,13 +32,35 @@ Content-Type: application/json
 {
   "revision": "main",
   "jobConfigs": {
-    "analyzer": {},
-    "advisor": {},
-    "scanner": {},
-    "evaluator": {},
-    "reporter": {},
-    "notifier": {}
-  }
+    "analyzer": {
+      "allowDynamicVersions": true,
+      "packageCurationProviders": [],
+      "skipExcluded": true
+    },
+    "advisor": {
+    },
+    "scanner": {
+    },
+    "evaluator": {
+    },
+    "reporter": {
+      "skipExcluded": true
+    },
+    "parameters": {
+      "ruleSet" :  "evaluator.rules.kts",
+      "useClearlyDefined": "true",
+      "recipientAddresses": "recipient@example.com",
+      "fossIdServer": "https://fossid.example.com/instance/",
+      "fossIdProjectName": "FossID-Projects",
+      "fossIdWaitForResult": "true",
+      "fossIdKeepFailedScans": "false",
+      "fossIdUseDeltaScans":  "true",
+      "fossIdDeltaScanLimit": "5",
+      "snippetReport": "true",
+      "fossIdFetchSnippetMatchedLines": "true"
```

Comment:
These are all Bosch internal parameters that require a special `.params.kts` script. If we want to have request templates for internal testing we should create them in an internal repository.

Reply:
Can we put the options from ORT `reference.yml` instead ?

Reply:
You could do that, but then that request would not help you for internal testing.